### PR TITLE
fix(signal2020): update flag description for profile

### PR DESCRIPTION
**Pull request by ktalebian**
* Update `--profile` description to distinguish from signal profile

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: src/commands/signal2020.js [hunk 1]
```diff
@@ -157,9 +158,12 @@ class Signal2020Command extends TwilioClientCommand {
   }
 }
 
+const baseFlags = { ...TwilioClientCommand.flags };
+baseFlags.profile.description = 'Shorthand identifier for your Twilio profile.';
```
Comment (childish-sambino) [inline src/commands/signal2020.js]: `s/Twilio/twilio-cli/`

**Review by childish-sambino — APPROVED** [review]

File: src/commands/signal2020.js [hunk 2]
```diff
@@ -157,9 +158,12 @@ class Signal2020Command extends TwilioClientCommand {
   }
 }
 
+const baseFlags = { ...TwilioClientCommand.flags };
```
Comment (dkundel) [inline src/commands/signal2020.js:160]: Can you add `Run twilio profiles:list for more info`

**Review by dkundel — CHANGES_REQUESTED** [review]

File: src/commands/signal2020.js (hunk 1 above)
Comment (ktalebian) [reply src/commands/signal2020.js]: Done

File: src/commands/signal2020.js (hunk 2 above)
Comment (ktalebian) [reply src/commands/signal2020.js:160]: Done

**Review by dkundel — APPROVED** [review]

**Pull request merged**